### DISCUSSION TITLE
First feedback

### DISCUSSION THREAD

**Discussion by HottonJoris** [Kaplas80/TF3#15] [General]
Hi ! Just tested TF3 with kiwami 2 plugins and it works like a charm ! Well done 💯

Here some "advice" for the documentation:
-  dotNet Runtime 6 required

Now I've a lot of translations to do 😂  I don't know if it is possible to export them like the TF2 allowed it.

**Comment by Kaplas80** [comment]
Thank you very much, I've just updated the readme.

Please, let me know if you find any bug or have any other suggestion.

**Comment by HottonJoris** [comment]
Just tested to rebuild assets, no issues, but in-game no translations at all :( 
-> file: (auth.bin)a01_0013.po  (first dialogues of the game)

(and other files like ui.text)

the command to rebuild gave me a folder **data** with db.par and ui.par  

Have any clues ?

**Reply by HottonJoris** [reply]
Ok I found my mistake :joy:

Work like a charm ! thx a lot again

**Reply by MaybeDaveyy** [reply]
Can you tell me what was the mistake, pls? I'm having the same issue and been trying to fix it, but I gave up :/
